docs/_templates/include/meetups/listing.jinja

```diff
@@ -35,6 +35,13 @@
               </div>
             {% endif %}
 
+            {# Other internet presence  #}
+            {% if meetup['website'] %}
```

Comment:
I wonder about doing this with `links` and allowing multiple?

Something like:

```
- links:
  - Meetup: https://meetup...
  - LinkedIn: https://linked..
```

That way you can control the link copy and allow multiple if needed?

Reply:
<ul>
{% for text, link in meetup['links'].items() %}
<li>
 <a href="{{ link }]">{{ text }}</a>
</li>
{% endfor %}
</ul>